src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,9 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+
+    for(let i = 0; i < employees.length; i++)
+    if (employeeName === employees[i].name)
+    return employees[i].role;
```

Comment:
I would try and put your return statement on the same line as the if statement here if you're not going to wrap it in `{}` eg `if (employeeName === employees[i].name) return employees[i].role`